_posts/2017-10-25-myget-by-the-numbers-2017.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "MyGet by the numbers 2017"
+date: 2017-11-03 13:00:00 +0200
+comments: true
+published: true
+categories: ["post"]
+tags: ["Development", "Stories"]
+author: "Xavier Decoster"
+---
+
+Since our [previous MyGet by the numbers](https://blog.myget.org/post/2016/07/28/myget-by-the-numbers.html) post in July 2016, MyGet has grown a lot! We've added support for two additional package managers: [Maven](https://blog.myget.org/post/2017/02/09/maven-packages-just-arrived-on-myget-early-access-preview.html) and [PHP Composer](https://blog.myget.org/post/2017/09/11/php-composer-packages-on-myget.html), bringing the total number of support package managers to 6 (*NuGet, npm, Bower, Visual Studio Extensions, Maven, PHP Composer*), and debugger symbols, of course.
+
+As MyGet has been growing, we've also been facing various challenges in terms of scalability. In the face of those challenges, we're proud on our uptime numbers and have been working hard on improving MyGet's scalability and reliability. We still have work to do to further improve performance which will receive great attention going forward. We figured it'd be an interesting story to share with you. Of course we can’t share all of our statistics, but it’s always fun to look at them. 
+
+## Shared Infrastructure
+
+MyGet is hosted on Microsoft Azure and has regional deployments in Europe and US. MyGet has a shared tenant, [www.myget.org](https://www.myget.org), home of the personal subscription plans (Free, Starter and Professional subscriptions). [MyGet Enterprise](https://www.myget.org/enterprise) tenants run on their own dedicated infrastructure, which may differ from what is outlined below.
+
+The shared tenant has its primary deployment in the West Europe region, and a secondary in US East. We use Azure Cloud Services to deploy both Web and Worker roles, and apply auto-scaling policies on both to ensure the service scales as needed.
+
+On average, the shared tenant has 4 Web roles and 4 worker roles deployed in each region, and scales in and out multiple times (minimum 2 instances per role, up to 8 during peak moments). Judging by the observed peak moments, all of us restore most packages in the morning (which is multiple times a day when running a global service).
+
+## Reliability
+
+Being developers ourselves, we know how frustrating it is when a service you use on a daily basis goes down. Even though we cannot offer an official SLA just yet, we can demonstrate historical uptime data on our [status page](https://www.myget.org/status). We invite you to go take a look if interested (and it lists uptime data for other endpoints too). 
```

Comment:
Ugh, last night was not helpful for these claims :-)

Reply:
We may need to come up with a custom status page one day so we can elaborate on the meaning of those endpoints a bit more... EU West package downloading was little affected, but querying v2 was on EU deployment.